pandemonium/server_payload.go

```diff
@@ -8,6 +8,9 @@ const (
 	PongOp          OpcodeType = "PONG"
 	RatelimitOp     OpcodeType = "RATE_LIMIT"
 	HelloOp         OpcodeType = "HELLO"
+	AuthenticatedOp OpcodeType = "AUTHENTICATED"
+	UserUpdateOP    OpcodeType = "USER_UPDATE"
+	PresenceUpdateOP OpcodeType = "PRESENCE_UPDATE"
```

Comment:
Has this been [formatted](https://go.dev/blog/gofmt)? It is built in to the VSCode extension.